nautobot_ssot/management/commands/elongate_interface_names.py

```diff
@@ -0,0 +1,50 @@
+"""Django Management command to update DCIM.Interface names."""
+from django.core.management.base import BaseCommand
+
+from nautobot.dcim.models import Device
+
+from netutils.interface import canonical_interface_name as netutils_elongate_interface_name
+
+
+class Command(BaseCommand):
+    """MGMT command to use netutils to update Nautobot Interface names to use long format."""
+
+    help = (
+        "Elongate DCIM.Interface names in Nautobot based off of netutils: "
+        "https://github.com/networktocode/netutils/blob/main/netutils/interface.py#L97"
```

Comment:
What about linking to the header for this in the docs versus a branch link with line number?

https://netutils.readthedocs.io/en/latest/dev/code_reference/interface/#netutils.interface.canonical_interface_name

Reply:
ya, I didn't like that when I did it, but couldn't think of something better. Your suggestion makes better sense.